owncloudApp/src/main/res/xml/settings_video_uploads.xml

```diff
@@ -35,13 +42,13 @@
         app:key="video_uploads_source_path"
         app:title="@string/prefs_camera_upload_source_path_title" />
     <ListPreference
-        app:iconSpaceReserved="false"
         app:defaultValue="NOTHING"
         app:dialogTitle="@string/prefs_camera_upload_behaviour_dialog_title"
         app:entries="@array/pref_behaviour_entries"
         app:entryValues="@array/pref_behaviour_entryValues"
+        app:iconSpaceReserved="false"
         app:key="video_uploads_behaviour"
-        app:useSimpleSummaryProvider="true"
-        app:title="@string/prefs_camera_upload_behaviour_title" />
+        app:title="@string/prefs_camera_upload_behaviour_title"
+        app:useSimpleSummaryProvider="true" />
```

Comment:
Attribute `negativeButtonText=""` not necessary here? If it isn't, maybe we can remove them from the other ListPreferences, but I would say it is

Reply:
Yes, it is needed if we do not want to show the cancel button in the dialog. So this is a glitch, good catch!